src/schema/notifications.ts

```diff
@@ -137,6 +137,10 @@ export const typeDefs = /* GraphQL */ `
     Attachments of the notification
     """
     attachments: [NotificationAttachment!]
+    """
+    Total number of avatars
+    """
+    numTotalAvatars: Int
```

Comment:
This change seems unrelated to the fix to me. Can you explain why it's needed in this PR?

Reply:
We have to add new field to Notification graphql type as well so that client can request it.